src/test/scala/chiselTests/util/SRAMSpec.scala

```diff
@@ -242,11 +242,29 @@ class SRAMSpec extends ChiselFlatSpec {
         numWritePorts = 0,
         numReadwritePorts = 1
       )
-      sramPath := sram.description.hierarchy
+      sramPath := sram.description.get.hierarchy
     }
     // TODO we need a way with ChiselSim to evaluate properties
     val chirrtl = emitCHIRRTL(new Top)
     chirrtl should include("output sramPath : Path")
     chirrtl should include("propassign sramPath, sram.description.hierarchy")
   }
+
+  it should "be possible to create an SramInterface Wire" in {
+    class Top extends Module {
+      val sramIntf = Wire(
+        new SRAMInterface(
+          memSize = 32,
+          tpe = Vec(4, UInt(8.W)),
+          numReadPorts = 0,
+          numWritePorts = 0,
+          numReadwritePorts = 1,
+          masked = false
+        )
+      )
+      sramIntf := DontCare
+    }
+    // should not give uninitialized errors for description properties
+    emitSystemVerilog(new Top)
```

Comment:
Does `DontCare` affect Properties? Would this test pass even if `hasDescription` were `true`?

Reply:
As it currently stands, `DontCare` does not appear to affect properties. Is this likely to change in the future? I should rewrite the test if so...

> Would this test pass even if hasDescription were true?

The test fails if `hasDescription` is set to `true`

Reply:
I am not sure if it will ever change, probably not so I think we're okay.